tox.ini

```
@@ -0,0 +1,34 @@
+[tox]
+envlist = py27,py35,py36,py37,pypy,pypy3,lint,docs,coverage-combine
```

Not too familiar with tox, but my understanding is that we probably don't want 2.7 here?

```suggestion
envlist = py35,py36,py37,pypy,pypy3,lint,docs,coverage-combine
```

I think there is also a reference on L30, but might be my lack of familiarity with tox.

---

Good catch, fixed.